### DISCUSSION TITLE
Using the plugin with Ionic Angular

### DISCUSSION THREAD

**Discussion by steven-fernandez** [HaylLtd/cordova-plugin-background-geolocation#104] [General]
I'm read all the documentation and examples and cannot find anything on setting up this plugin in Angular / typescript - e.g: import  ... 
I've tried many variations but I can't run it, I just get errors.

Has anyone managed to get this plugin to work in an Ionic / Angular project?

Any tips is greatly appreciated.

**Comment by HarelM** [comment]
https://github.com/IsraelHikingMap/Site/blob/b96803844c2bcebb1a50bf03b5fab96232e9f197/IsraelHiking.Web/src/application/services/geo-location.service.ts
The following is what I use in my app...
